src/app/derive/derive.component.ts

```diff
@@ -37,7 +35,8 @@ export class DeriveComponent implements OnInit {
     private identityService: IdentityService,
     public globalVars: GlobalVarsService,
     private backendApi: BackendAPIService,
-    private activatedRoute: ActivatedRoute
+    private activatedRoute: ActivatedRoute,
+    public truncateAddress: TruncateAddress
```

Comment:
```suggestion
    private activatedRoute: ActivatedRoute
```
We don't need to import truncate address